joomla/joomla-cms
# [com_tags] - Add count and link to tagged items

redo of  #8975 .
#### Summary of Changes

see comment https://github.com/joomla/joomla-cms/pull/8975#issuecomment-217636135
#### Testing Instructions

see #8975
You can select the tag "content type"  alias article, contact, newsfeed, weblink ect
![staging administration tags manager](https://cloud.githubusercontent.com/assets/181681/12536158/c17f2904-c29c-11e5-812a-82649b5075ec.png)
and have a better idea where tags  are located
![test administration tags manager2](https://cloud.githubusercontent.com/assets/181681/12536171/ff85adc2-c29c-11e5-927c-1d0a5b9b4e23.png)
